tests/executables_src/testAsyncNDRegisterAccessor.cpp

```diff
@@ -41,8 +37,8 @@ BOOST_AUTO_TEST_CASE(Test_setWriteAccessor) {
 
   BOOST_CHECK(int(asyncAccessor) != 43);
 
-  asyncAccessor = 43;
-  asyncAccessor.write();
+  syncAccessor = 43;
+  syncAccessor.write();
 
   BOOST_CHECK_EQUAL(d.read<int>("MODULE0/INTERRUPT_TYPE"), 43);
 }
```

Comment:
These lines were the actual test. By changing it to symcAccessor you are only testing that DUMMY_WRITEABLE is working.